### PR TITLE
fixbug: role init with is_human=True was not work

### PR DESCRIPTION
**Pull request by kkdev163**
bugfix:  role init with is_human=True was not work

### CODE REVIEW

**Comment by kkdev163** [comment]
When I was browsing the "Tutorials-Human Engagement" section, I encountered an issue where human intervention was not effective. I hope this issue can be fixed in a timely manner. I am a beginner in Python, so if there is a better way to fix this problem, please advise.  @geekan

File: metagpt/roles/role.py [hunk 1]
```diff
@@ -166,6 +166,9 @@ def __init__(self, **data: Any):
         Role.model_rebuild()
         super().__init__(**data)
 
+        if data.get("is_human"):
```
Comment (garylin2099) [inline metagpt/roles/role.py:169]: Good catch. Can use `if self.is_human` here

Comment (kkdev163) [reply metagpt/roles/role.py]: Thanks, please review my latest commit

**Comment by garylin2099** [comment]
According test run:

1. Unittest looks fine (you can ignore the errors, the errors are not related to your code)
2. Pre-commit checks fail. There is a formatting issue, potentially the import order in your test_role.py. Check https://github.com/geekan/MetaGPT/blob/main/.pre-commit-config.yaml, run the command `pre-commit run --all-files` and commit your format change

**Comment by kkdev163** [comment]
> pre-commit run --all-files and commit your format change

Thanks for giving guide. Please help me review once more.

**Comment by garylin2099** [comment]
LGTM

**Pull request merged**